src/TesApi.Tests/TerraApiStubData.cs

```diff
@@ -6,10 +6,14 @@ namespace TesApi.Tests;
 
 public class TerraApiStubData
 {
-    public string ApiHost => "https://landingzone.host";
+    public string LandingZoneApiHost => "https://landingzone.host";
+    public string WsmApiHost => "https://wsm.host";
+
     public string ResourceGroup => "mrg-terra-dev-previ-20191228";
```

Comment:
is this supposed to be here?

Reply:
Yeah this is stub/fake data for testing and to validate parsing